src/core/annotation.js

```diff
@@ -951,6 +951,13 @@ class WidgetAnnotation extends Annotation {
     });
     data.fieldValue = this._decodeFormValue(fieldValue);
 
+    const defaultFieldValue = getInheritableProperty({
+      dict,
+      key: "DV",
+      getArray: true,
+    });
+    data.defaultFieldValue = this._decodeFormValue(defaultFieldValue);
```

Comment:
Please add unit test coverage for this property. I'd recommend to simply extend the existing unit tests for `fieldValue` to also check `defaultFieldValue`, and set a value for it per annotation type tested.